helpers/securewebhook/securewebhook.go

```diff
@@ -27,3 +46,38 @@ func GetRequestBody(s *Settings) ([]byte, error) {
 	}
 	return b, nil
 }
+
+// ConvertPublicKeyBase64ToECDSA takes a base64 ECDSA public key and converts it into the ECDSA Public Key type
+func ConvertPublicKeyBase64ToECDSA(base64PublicKey string) (*ecdsa.PublicKey, error) {
+	pk, err := base64.StdEncoding.DecodeString(base64PublicKey)
+	if err != nil {
+		return nil, err
+	}
+
+	publicKey, err := x509.ParsePKIXPublicKey(pk)
+	if err != nil {
+		return nil, err
+	}
+
+	return publicKey.(*ecdsa.PublicKey), nil
+}
+
+// VerifySignature uses the ECDSA publicKey and verifies received payload and signature
+func VerifySignature(publicKey *ecdsa.PublicKey, payload []byte, signature, timestamp string) (bool, error) {
```

Comment:
should we finalize the order of arguments passed into this function? i'll match whatever we decide in sendgrid-java for some level of consistency

Reply:
I am good with this order.